test/integration_tests/conftest.py

```diff
@@ -40,6 +40,12 @@ def ctc_decoder():
     "fr": "20121212-0900-PLENARY-5-fr_20121212-11_37_04_10.flac",
     "it": "20170516-0900-PLENARY-16-it_20170516-18_56_31_1.flac",
 }
+_MIXTURE_FILES = {
+    "speech_separation": "mix_3729-6852-0037_8463-287645-0000.wav",
+}
+_EXPECTED_TENSORS = {
+    "speech_separation": "expected_3729-6852-0037_8463-287645-0000.pt",
```

Comment:
Please replace with this regular audio format.

Reply:
The reason why I used tensor before is the output waveform has large values, after saving it to audio file it causes clipping.

Let me try to save the normalized waveform to see if the peak frequency test can pass.

Reply:
In that case, I think it makes more sense to compare against the ground truth audio, and apply threshold to the SNR between original reference and result.

Reply:
So rather than uploading the estimated waveforms, we can upload the ground truth waveforms and compute the SNR scores then compare it with a given values? Makes sense to me.
Can we use third-party library for metric computation, for example, `mir_eval`?